views/partials/beerpartials.hbs

```diff
@@ -0,0 +1,6 @@
+<h1>Ironbeers</h1>
```

Comment:
This Ironbeers header tag will appear everywhere and also for each beer. Better to move it out.